FindYourRecipe/FindYourRecipe.Web/wwwroot/sample-data/weather.json

```diff
@@ -0,0 +1,28 @@
+﻿[
+  {
```

Comment:
This file can be removed as it is useless for you project considering that you do not manage weather information.